includes/legacy/abstract-wc-legacy-order.php

```diff
@@ -454,7 +454,7 @@ public function __get( $key ) {
 			return $this->{"get_{$key}"}();
 		} else {
 			return get_post_meta( $this->get_id(), '_' . $key, true );
-		}
+		} // End if().
```

Comment:
@bor0 better use `phpcs --standard=./phpcs.ruleset.xml` or `grunt phpcs` (but this will try for all files).